lib/utils/helpers.ts

```diff
@@ -187,3 +187,27 @@ export function getSubShellCommand(command: string) {
 		};
 	}
 }
+
+export function retry<T>(
+	func: () => T,
+	times: number,
+	label: string,
+	delayMs = 1000,
+	factor = 2,
+	count = 0,
+): Bluebird<T> {
+	let promise = Bluebird.try(func);
+	if (count < times) {
+		promise = promise.catch((err: Error) => {
+			const delay = factor ** count * delayMs;
```

Comment:
Associativity for the operators used in this statement are not consistent. It's a lot nicer to make this explicit:
```typescript
const delay = (factor ** count) * delayMS;
```
That said, it'd also be nice to make it explicit that `factor` is a backoff scaler, perhaps `backoffScaler` rather than `factor`?

Reply:
Renamed. The parentheses are automatically removed by `prettier`.

Reply:
Thanks. Prettier strikes again. :(